jenkins/scripts/notify_slack.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+# get build status and branch
+# send notification
+
+STATUS=$1 #job status
```

Comment:
Instead of using doing status check like this, you can make use of `post` function in Jenkinsfile.

Reply:
Can you please expound on this @HarithJ with reference to the `Jenkinsfile`.

Reply:
I am using one script to notify Slack of the build result per stage, the script requires as the first argument, the status of the job which I pass from the Jenkins at the post step.